octobot/evaluator_factory.py

```diff
@@ -50,6 +53,19 @@ def __init__(self, octobot):
                                              exchange_configuration.symbols,
                                              exchange_configuration.time_frames)
         await init_evaluator_chan_logger()
+        # Start service feeds now that evaluators registered their feed requirements
+        await self._start_service_feeds()
+
+    def _initialize_service_feeds(self):
```

Comment:
I don't see any evaluator_factory methods or attributes used by service methods. So I don't think It's a good idea to put services initialization here. We should initialize it in a different file.

Reply:
I put it there because service feeds are used by evaluators and should be created and started at a specific time to allow evaluators to register first. We can put it in a dedicated file and call it from ```evaluator_factory.py```

Reply:
Yes I think we should put this in another file. But I don't know if it's better to call in from `evaluator_factory.py` or `octobot.py`.